README.md

```diff
@@ -1,2 +1,52 @@
 # ft_transcendence
-🏓 Multiplayer pong tournaments
+🏓 Play pong online with your friends in real time
+
+# 🤝 Contributing
+
+If you want to contribute to this project you may follow a set of rules.
+
+### 1. Create a new issue or task:
+
+- With every contribution there must be a related issue/task, so if you
+don't have any, create a new one on the project board with some
+detailed info about what do you want to accomplish.
+
```

Comment:
Any naming convention for this? #number, FIX, new feature, ...?

Reply:
Nice idea! I'd say just adding a brief description about what is about and, then, tag appropiately the issue with the type (feature, task, bug etc.)

Do all you agree? If so I'll update the doc